src/app/pages/home/home.component.html

```diff
@@ -1,3 +1,7 @@
+<header>
```

Comment:
since all your header components are in the navbar, put the header tag inside it